src/test_report.py

```diff
@@ -164,7 +127,7 @@ def run_loop(self):
 
     def run_from_id(self, node_id, dump, send):
```

Comment:
The same comment as for `run_loop` method applies here, but instead of `cmd_run` I'd put it in the `cmd_single`